.travis.yml

```diff
@@ -88,7 +91,7 @@ matrix:
         - RUSTFLAGS='-C debuginfo=0' cargo test --target asmjs-unknown-emscripten
 
     - name: "Linux, nightly, docs"
-      rust: nightly
+      rust: nightly-2020-09-08
```

Comment:
I am not sure if it's a good idea to pin Nightly. Even if in future some change will break our build, I think it's better to find about it sooner than later.

Reply:
Would you want to mark all of the nightly builds as non-blocking then? We use nightly for:
  - a Linux build
  - a Mac build
  - cross compiling
  - All of our WASM targets

Reply:
I generally agree with this point though. I think we should move as much as we can out of the nightly tests into the stable tests. Then, mark all the stable tests as non-blocking.

Reply:
Did we have problems with nightly builds previously? If I had to chose, I would prefer making those jobs non-blocking.

Reply:
We did, a nightly didn't have some component we needed (rustfmt maybe, but that's already non-blocking).

I agree that we should have them be blocking, should be fixed now.

Reply:
BTW do we have cron jobs enabled in Travis?

cc @dhardy

Reply:
Looking at https://travis-ci.org/github/rust-random/getrandom/builds/714877349 and https://travis-ci.org/github/rust-random/getrandom/builds/724230064 they are monthly cron jobs